decrypt/decrypt.go

```diff
@@ -85,7 +89,7 @@ func Data(data []byte, format string) (cleartext []byte, err error) {
 
 	// Use a constant-time MAC tag comparison to avoid timing attacks: https://codahale.com/a-lesson-in-timing-attacks/
 	if subtle.ConstantTimeCompare(computedMacBytes, originalMacBytes) != 1 {
-		return nil, fmt.Errorf("Failed to verify data integrity. expected mac %q, got %q", originalMac, mac)
+		return nil, errMacMismatch
```

Comment:
Well it's generally not a good idea to tell the client what value they should have used for a MAC tag, as that is their goal when trying to create a forgery. In the current design this is safe, as the MAC is a publicly computable hash, but in most MAC designs it wouldn't be safe as the computed value should remain secret.

It seemed of marginal value to actually include the particular 128-hex-digit pseudorandom strings that were not equal to each other, as an incorrect MAC tag/hash is usually different in most or all digits rather than off-by-one or anything simple.